doc/source/examples/cd_clf_adult.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>Nitpick: I think the first warning is something that we could clean up elsewhere too, i.e. <code>train_size</code>  has a default value but <code>n_folds</code>  doesn't, however, it doesn't feel natural that a warning should be raised because a user provides <code>n_folds</code>  argument just because <code>train_size</code>  has a default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>